agenta-web/src/components/Evaluations/Evaluations.tsx

```diff
@@ -190,8 +189,23 @@ export default function Evaluations() {
         llmAppPromptTemplate?: string,
     ) => {
         const postData = async (url = "", data = {}) => {
-            const response = await axios.post(url, data)
-            return response.data
+            const response = await fetch(url, {
```

Comment:
@bekossy If you don't want axios to handle the error, you can pass `_ignoreError` property in the request config. Please see the `api.ts` file for examples. We should unify our Network calls to be sent only through axios in the codebase.